<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DFTSettingDialog</class>
 <widget class="QDialog" name="DFTSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="2,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_left" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0,1,0">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButton_open">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_save">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>476</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="GtPlot" name="widget_plot_amp" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="label_amp">
        <property name="text">
         <string>amplitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="GtPlot" name="widget_plot_phase" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="label_phase">
        <property name="text">
         <string>phase</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget_right" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_torque_factor">
          <property name="text">
           <string>torque factor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_tqr_factor"/>
        </item>
        <item>
         <widget class="QLabel" name="label_torque_factor_2">
          <property name="text">
           <string>Nm/Arms</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_current_command">
          <property name="text">
           <string>current command</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_current_unit">
          <item>
           <property name="text">
            <string>mA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_velocity_feedback">
          <property name="text">
           <string>velocity feedabck</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_velocity_unit">
          <item>
           <property name="text">
            <string>RPM</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>rad/s</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RPS</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_velocity_feedback_2">
        <property name="text">
         <string>inertia estimation</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_inertia_estimate">
          <property name="text">
           <string>start frequency</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_start_fre">
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>40</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>60</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>70</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>80</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>90</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>100</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_torque_factor_3">
          <property name="text">
           <string>HZ</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_inertia_estimate_2">
          <property name="text">
           <string>end frequency</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_end_fre">
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>40</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>60</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>70</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>80</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>90</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>100</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_torque_factor_4">
          <property name="text">
           <string>HZ</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_inertia_estimate_3">
          <property name="text">
           <string>result</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_inertia_result"/>
        </item>
        <item>
         <widget class="QLabel" name="label_torque_factor_5">
          <property name="text">
           <string>10^-6 kg.m^2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>270</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_calculate">
        <property name="text">
         <string>calculate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GtPlot</class>
   <extends>QWidget</extends>
   <header>gtplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
